README.md

```diff
@@ -1 +1,40 @@
-# techinical-documentation-review
\ No newline at end of file
+# Techinical documentation project review
+
+> This project display the technical documentation 
+> of a Java programming language, and it shows that basic 
+> parts that can be included in any other technical documentation. 
+
+## Built With
+
+- HTML 
+- CSS
+
+## Getting Started
+To get a local copy, just follow the steps That I am going to mention bellow:
+
+### Prerequisites
+
+### Setup
+To get this project in your local machine, follow these steps:
+- Open terminal 
+- clone the repository :https://github.com/ishimwezachee/techinical-documentation-review.git
+- Cd in the project folder
+- run the project in your browser
+
+### install
+ - run npm install to install the required linters
+
+ ### tests
+ - run the project in the browsers 
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@zacheeIshimwe](https://github.com/ishimwezachee)
+- LinkedIn: [@zacheeIshimwe](https://www.linkedin.com/in/zachee-ishimwe-ab952a119/)
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Well done, the documentation is official and it explains what your project is all about.